src/main/java/org/example/HibernateSessionFactoryUtil.java

```diff
@@ -0,0 +1,33 @@
+package org.example;
+
+
+import org.example.model.Employee;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+
+import java.lang.module.Configuration;
+
+public class HibernateSessionFactoryUtil {
+    private static SessionFactory sessionFactory;
+
+    private HibernateSessionFactoryUtil() {
+    }
+
+
+    public static SessionFactory getSessionFactory() {
+        if (sessionFactory == null) {
+            try {
+
+                Configuration configuration = new Configuration().configure();
+                configuration.addAnnotatedClass(Employee.class);
+                StandardServiceRegistryBuilder builder = new StandardServiceRegistryBuilder().applySetting(configuration.getProperties());
+                sessionFactory = configuration.buildSessionFactory(builder.build());
```

Comment:
Нужно подправить конфигурацию:

```suggestion
                Configuration configuration = new Configuration().configure();
                configuration.addAnnotatedClass(Employee.class);
                configuration.addAnnotatedClass(City.class);
                sessionFactory = configuration.buildSessionFactory();
```